src/elements/toggle-check/toggle-check.visual.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { html } from 'lit';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualDiffFocus,
+  visualRegressionFixture,
+} from '../core/testing/private.js';
+
+import '../icon.js';
+import '../title.js';
+import './toggle-check.js';
+
+describe(`sbb-toggle-check`, () => {
+  let root: HTMLElement;
+
+  const longLabel = `For this example we need a very long label, like lorem ipsum dolor sit amet, consectetur adipiscing elit.
+  Cras nec dolor eget leo porttitor ultrices. Mauris sed erat nec justo posuere elementum.
+  In pharetra ante vel fringilla tincidunt. Fusce congue accumsan arcu dictum porttitor.
+  Pellentesque urna justo, lacinia at velit eu, sagittis tempus nibh.
+  Quisque vitae massa et turpis fermentum tristique.`;
+
+  const sizeCases = { size: ['xs', 's', 'm'] };
+
+  const cases = {
+    ...sizeCases,
+    label: ['Label', 'Long label'],
+  };
+
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    describeEach(cases, ({ size, label }) => {
+      beforeEach(async function () {
+        root = await visualRegressionFixture(html`
+          <sbb-toggle-check size=${size as 'xs' | 's' | 'm'}>
+            ${label !== 'Long label' ? label : longLabel}
+          </sbb-toggle-check>
+        `);
+      });
+
+      it(
+        visualDiffDefault.name,
+        visualDiffDefault.with((setup) => {
+          setup.withSnapshotElement(root);
+        }),
+      );
+    });
+
+    describe('checked', () => {
+      for (const state of [visualDiffDefault, visualDiffFocus]) {
+        it(
+          state.name,
+          state.with(async (setup) => {
+            await setup.withFixture(html` <sbb-toggle-check checked> Label </sbb-toggle-check> `);
+          }),
+        );
+      }
+    });
+
+    describe('without label', () => {
```

Comment:
For test cases with only one 'it' you dont need a describe. You can directly set it as title of 'it' also state (default here) is not needed